mne/decoding/ssd.py

```diff
@@ -39,35 +39,36 @@ class SSD(BaseEstimator, TransformerMixin):
         Filtering for the frequencies of non-interest.
     reg : float | str | None (default)
         Which covariance estimator to use.
-        If not None (same as 'empirical'), allow regularization for
-        covariance estimation. If float, shrinkage is used
-        (0 <= shrinkage <= 1). For str options, reg will be passed to
-        method to :func:`mne.compute_covariance`.
+        If not None (same as 'empirical'), allow regularization for covariance
+        estimation. If float, shrinkage is used (0 <= shrinkage <= 1). For str
+        options, reg will be passed to method :func:`mne.compute_covariance`.
     n_components : int | None (default None)
         The number of components to extract from the signal.
-        If n_components is None, no dimensionality reduction is applied.
+        If None, the number of components equal to the rank of the data are
+        returned (see :attr:`rank`).
```

Comment:
Sphinx has trouble resolving this so let's just use code mode for now
```suggestion
        returned (see ``rank``).
```